crates/Cargo.toml

```diff
@@ -1,19 +1,12 @@
 [workspace]
-members = [
-    "alloc",
-    "flipperzero",
-    "sys",
-    "rt",
-    "test",
-    "test/macros",
-]
+members = ["alloc", "flipperzero", "sys", "rt", "test", "test/macros"]
 resolver = "2"
 
 [workspace.package]
 version = "0.11.0"
 description = "Rust for Flipper Zero"
 edition = "2021"
-rust-version = "1.70.0"
+rust-version = "1.81.0"
```

Comment:
Is this the required MSRV to be able to build with stable Rust?

Reply:
It is, at least since `error_in_core` was stabilized in it.